packages/office-ui-fabric-react/src/components/Panel/Panel.base.tsx

```diff
@@ -190,43 +210,49 @@ export class PanelBase extends BaseComponent<IPanelProps, IPanelState> implement
   }
 
   public open() {
-    if (!this.state.isOpen) {
-      this.setState(
-        {
-          isOpen: true,
-          isAnimating: true
-        },
-        () => {
-          this._async.setTimeout(this._onTransitionComplete, 200);
-        }
-      );
+    if (this.props.isOpen !== undefined) {
+      return;
+    }
 
-      if (this.props.onOpen) {
-        this.props.onOpen();
-      }
+    if (this.isActive) {
+      return;
     }
+
+    this._clearExistingAnimationTimer();
```

Comment:
Why do you clearExistingAnimationTimer here? I don't think you need to since componentDidUpdate will get called after the setstate is complete. Same with close below.

Reply:
Good catch. This was indeed redunant.